jujugui/static/gui/src/app/app.js

```diff
@@ -684,6 +688,23 @@ YUI.add('juju-gui', function(Y) {
     },
 
     /**
+      Sets up the UIState instance on the app
+
+      @method _setupUIState
+      @param {Boolean} sandbox
+      @param {String} baseUrl
+    */
+    _setupUIState: function(sandbox, baseUrl) {
+      this.state = new models.UIState({
+        // Disallow routing to inspectors if we are in sandbox mode; the
+        // model to be inspected will not be available.
+        allowInspector: !sandbox,
+        baseUrl: baseUrl || '',
+        dispatchers: {}
```

Comment:
Maybe a comment here to why we don't set any initial dispatchers would be helpful.